pkgs/top-level/python-packages.nix

```
@@ -5271,6 +5271,8 @@ in {
 
   prompt_toolkit = callPackage ../development/python-modules/prompt_toolkit { };
 
+  prompt_toolkit1 = callPackage ../development/python-modules/prompt_toolkit/1.nix { };
```

```suggestion
  prompt-toolkit1 = callPackage ../development/python-modules/prompt_toolkit/1.nix { };
```

Normally it should be like this but I am not sure if that would be confusing if the normal version is the other way around.